trade-finance-manager-api/src/v1/helpers/get-submission-date.api-test.js

```diff
@@ -0,0 +1,18 @@
+const getSubmissionDate = require('./get-submission-date');
+const MOCK_GEF_AIN_DEAL = require('../__mocks__/mock-gef-deal');
+const MOCK_GEF_MIA_DEAL = require('../__mocks__/mock-gef-deal-MIA');
+const MOCK_GEF_MIN_DEAL = require('../__mocks__/mock-gef-deal-MIN');
+
+describe('Ascertain deal submission date', () => {
+  it('Should return `submissionDate` as deal is an AIN', () => {
+    expect(getSubmissionDate(MOCK_GEF_AIN_DEAL)).toEqual(1626169888809);
```

Comment:
not a blocker, but could you change the '1626169888809' and all other numbers to MOCK_GEF_AIN_DEAL.manualInclusionNoticeSubmissionDate incase anyone changes the mocks.  Can be done later

Reply:
Updated.